### PR TITLE
Support django 1.10

### PR DESCRIPTION
**Pull request by brad**
@orcasgit/orcas-developers Please review. This should support Django 1.10 and be backward compatible down to Django 1.8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,12 @@
...
-  - pip install coveralls
-  - pip install -r requirements/test.txt
-script:
-  - tox -e $TOX_ENV
-  - coverage report -m
-after_success:
-  coveralls --verbose
+  - pip install coveralls tox-travis
+script: tox
+after_success: coveralls
```
Comment (grokcode) [inline .travis.yml]: @brad 👍  I like the simplifications here.

**Comment by grokcode** [comment]
@brad It looks like there might be a problem with the tests here.

**Comment by brad** [comment]
@grokcode OH shoot, I missed that, thanks

**Comment by grokcode** [comment]
Looks great! :shipit:

**Pull request merged**